cms/templates/js/certificate-web-preview.underscore

```diff
@@ -7,7 +7,7 @@
 <a href=<%- certificate_web_view_url %> class="button preview-certificate-link" rel="noopener" target="_blank">
     <%- gettext("Preview Certificate") %>
 </a>
-<button class="button activate-cert" onclick="mixpanel.track('certificate_activate',{'service':'studio', 'screen_name':'certificates', 'organization':<%-platform_name%>})">
+<button class="button activate-cert" onclick="mixpanel.track('certificate_activate', {'service': 'studio', 'screen_name': 'certificates', 'organization': '${ platform_name }'})">
```

Comment:
Can we make this change from themes?
https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/create_theme.html

Reply:
Ok sure but should I remove the existing code from this file as well? Because there were already changes in it